Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -340,3 +340,37 @@
   - type: HeldSpeedModifier
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitScaf
+
+# Tactical Maid Hardsuit
+
+- type: entity
+  parent: ClothingOuterHardsuitBase
+  id: ClothingOuterHardsuitTacticalMaid
+  name: tactical maid hardsuit
+  description: Multiple coats of stain and dust resistant compounds all built into a single suit.
```

This is a silly item, why not give it a silly description?
```suggestion
  description: Multiple coats of stain resistant compounds all built into a single suit. Aww, it even has an apron.
```

---

"Layers of stain resistant alloys built into a single suit. It even comes with a reinforced apron!"

---

Sure.